patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -272,6 +272,17 @@
  	public static bool allChestStackHover;
  	public static bool inventorySortMouseOver;
  	public static float GraveyardVisualIntensity;
+@@ -868,6 +_,10 @@
+ 	public static short[] tileGlowMask = new short[TileID.Count];
+ 	public static bool[] tileContainer = new bool[TileID.Count];
+ 	public static bool[] tileSign = new bool[TileID.Count];
++	/// <summary>
++	/// A check to fix multiplayer support, used in TileLoader.IsModMusicBox
++	/// </summary>
++	public static bool[] tileIsModMusicBox = new bool[TileID.Count];
```

Comment:
I would suggest moving this to `TileID.TML.cs`, in the `Sets` class. initialize it similar to other default-to-false arrays.
And change the doc to something more descriptive for modders, you can turn the current summary into a single comment instead as it's mainly tml-internal stuff

Reply:
👍

Reply:
👍